python/mp/effects/cop_lights.py

```diff
@@ -0,0 +1,28 @@
+import copy
+from mp import color as _color
+from mp.effects import effect, bin, looper, strobe
+
+class CopLights(effect.Effect):
+    """
+    """
+
+    # Wish there were a better way than requiring this every time
+    dm = copy.deepcopy(effect.Effect.dm)
+
+    def __init__(self, bead_set, color=_color.Color(), direction=1, **kwargs):
+        super().__init__(name="cop_lights", bead_set=bead_set, color=color, **kwargs)
+
+        self.bin = bin.Bin(bead_set)
```

Comment:
CopLights uses 4 loopers and a strobe effect to create the lighting impression of a cop car.